src/pages/PhotoPage.tsx

```diff
@@ -9,32 +9,62 @@ type PhotoPageProps = {
 }
 
 function PhotoPage({ toggleHeader }: PhotoPageProps) {
+  const [windWidth, setWindWidth] = useState(window.innerWidth)
+  function isNavShowing() {
+    setWindWidth(window.innerWidth)
+  }
+
   useEffect(() => {
     toggleHeader(false)
-    return () => toggleHeader(true)
-  }, [toggleHeader])
+    window.addEventListener("resize", isNavShowing)
+    if (windWidth > 900) {
+      toggleHeader(true)
+    } else {
+      toggleHeader(false)
+    }
+    return () => {
+      toggleHeader(true)
+      window.removeEventListener("resize", isNavShowing)
+    }
+  }, [toggleHeader, windWidth])
 
-  const { photoId } = useParams<{ photoId: string }>()
-  const { photoQuery } = useApi()
+  const { albumId, photoId } = useParams<{ albumId: string; photoId: string }>()
 
-  const { data, error, isError, isLoading } = photoQuery(photoId)
-  console.log(data)
+  const { photoInAlbum, photoQuery } = useApi()
```

Comment:
Please add `Query` suffix to `photoInAlbum` to keep it in the same style as the rest.